packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -133,10 +130,26 @@ export default class SearchBox extends Base {
    * Invoked in response to a user clicking inside of the searchbox input.
    */
   clickExposed() {
-    const searchBoxClickedEvent = new CustomEvent(SEARCHBOX_EVENT.SEARCHBOX_CLICK, { bubbles: true });
+    const searchBoxClickedEvent = this.createCustomEvent(SEARCHBOX_EVENT.SEARCHBOX_CLICK);
     this.dispatchEvent(searchBoxClickedEvent);
   }
 
+  /**
+   * Returns an event with a standard structure.
+   *
+   * @param type The type (or name) of the event to be emitted.
+   * @param detail A payload to be sent with the event.
+   */
+  createCustomEvent(type: string, detail?: object) {
```

Comment:
Should this have a return type of `CustomEvent`?

Reply:
Come to think of it, this would be better typed as the following:

```suggestion
  createCustomEvent<T>(type: string, detail?: T): CustomEvent<T> {
```

Reply:
True! Good catch to you both.